backup.js

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env node
+
+const { exec } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const arg = require('arg');
+
+/* eslint-disable no-console no-process-env */
+
+/** *****************************************
+ * ARGUMENT PARSING
+ ****************************************** */
+
+const args = arg({
+	// Types
+	'--help': Boolean, // show the help
+	'-h': '--help',
+
+	'--path': String, // export path or directory to import from
+	'-p': '--path',
+
+	'--collection': [String], // collection(s) to import/export
+	'-c': '--collection',
+
+	'--username': String, // Mongo Username
+	'-U': '--username',
+
+	'--password': String, // Mongo Password
+	'-P': '--password',
+
+	'--database': String, // Mongo Database
+	'-D': '--database',
+
+	'--url': String, // Mongo Host String (ex. localhost:27017)
+	'-H': '--url',
+
+	'--pretty': Boolean,
+	'-b': '--pretty',
+}, {
+	permissive: true,
+	argv: process.argv.slice(2),
+});
+
+if (args['--help']) {
+	console.log(`Usage: node backup.js [opts] <export|import>
+
+OPTIONS:
+	--help (-h)        Show this help.
+
+	--path (-p)        export path or directory to import from
+	--collection (-c)  collection(s) to import/export, can be used multiple times
+
+	--username (-U)    Mongo Username
+	--password (-P)    Mongo Password
+	--database (-D)    Mongo Database
+	--url (-H)         Mongo Host String (ex. localhost:27017)
+
+	--pretty (-b)      Pretty Print (only on export)
+`);
+	process.exit(0);
+}
+
+/** *****************************************
+ * CONFIG
+ ****************************************** */
+
+const getTimestamp = () => {
+	const d = new Date();
+	return `${d.getUTCFullYear()}_${
+		(`0${d.getUTCMonth() + 1}`).slice(-2)}_${
+		(`0${d.getUTCDate()}`).slice(-2)}_${
+		(`0${d.getUTCHours()}`).slice(-2)}_${
+		(`0${d.getUTCMinutes()}`).slice(-2)}_${
+		(`0${d.getUTCSeconds()}`).slice(-2)}`;
+};
+
+const CONFIG = {
+	BASE_PATH: process.env.BASE_PATH || 'backup/',
+	get BACKUP_PATH_IMPORT() {
+		return path.join(this.BASE_PATH, args['--path'] || 'setup');
+	},
+	get BACKUP_PATH_EXPORT() {
+		return path.join(this.BASE_PATH, args['--path'] || getTimestamp());
+	},
+	MONGO: {
+		HOST: process.env.MONGO_HOST || '127.0.0.1',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of process.env.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5305080)